taskcluster/kinds/docker-image/kind.yml

```diff
@@ -21,6 +21,8 @@ tasks:
         parent: run-task
     fetch:
         symbol: I(fetch)
+    image-builder:
```

Comment:
The task name becomes `build-docker-image-image-builder`. Any way we could make the name a bit more sensible?

Reply:
Yeah it's weird, but I'd vote we leave it. Otherwise we no longer follow the `<kind>-<name>` format, which I'd argue is even more confusing.